src/Microsoft.Graph.Core/Models/Date.cs

```diff
@@ -10,7 +10,6 @@ namespace Microsoft.Graph
     /// <summary>
     /// Custom Date model for serialization
     /// </summary>
-    [JsonConverter(typeof(DateConverter))]
     public class Date
```

Comment:
do we still need this model since it's going to be in abstractions now?

Reply:
No, we don't. 
I had already captured this in the PR description though and my hope is to address this in a separate PR once https://github.com/microsoft/kiota/issues/1004 is closed. This is to avoid any double work from any incorrect assumptions based on any implementation quicks that may arise from that work.

Reply:
thanks for clarifying!